src/XMakeBuildEngine/Evaluation/LazyItemEvaluator.IncludeOperation.cs

```diff
@@ -58,6 +58,19 @@ protected override ICollection<I> SelectItems(ImmutableList<ItemData>.Builder li
                     }
                 }
 
+                var excludePatternsForGlobs = new Lazy<ISet<string>>(() =>
```

Comment:
Why would that be an improvement? The lazy setting before the first access vs using the Lazy seem to be doing the same thing (except creating that extra object for Lazy)